_partials/_devops-cli-global-flags.md

```diff
@@ -3,9 +3,9 @@
 |-------------------------------|-------------------|-----------------------------------------------------------------------------|
 | `--analytics`                 | `true`            | Set to `false` to disable usage analytics                                   |
 | `--color `                    | `true`            | Set to `false` to disable colored output                                    |
-| `--config-dir` string         | `.config/tiger`  | Set the directory that holds `config.yaml`                                  |
+| `--config-dir` string         | `~/.config/tiger` | Set the directory that holds `config.yaml`                                  |
```

Comment:
The path has to include `~` to be accurate. Otherwise it looks like it's putting it in your current working directory, rather than your $HOME directory.